static/js/components/dashboard/CourseDescription_test.js

```diff
@@ -23,32 +24,35 @@ import {
 } from '../../constants';
 
 describe('CourseDescription', () => {
-  let getElements = (renderedComponent) => {
-    let title = renderedComponent.find(".course-title");
-    let titleLink = title.find("a");
-    let details = renderedComponent.find(".details");
-    return {
-      titleText: title.text(),
-      titleLink: titleLink,
-      detailsText: details.text()
-    };
-  };
-
-  it('shows the course title and a link to view the course on edX', () => {
+  let getElements = (renderedComponent) => ({
+    titleText: renderedComponent.find(".course-title").text(),
+    edxLink: renderedComponent.find("a.view-edx-link"),
+    contactLink: renderedComponent.find("a.contact-link"),
+    detailsText: renderedComponent.find(".details").text()
+  });
+
+  let renderCourseDescription = (courseRun, courseTitle, canContactCourseTeam) => (
+    shallow(
+      <CourseDescription
+        courseRun={courseRun}
+        courseTitle={courseTitle}
+        canContactCourseTeam={R.isNil(canContactCourseTeam) ? true : canContactCourseTeam}
```

Comment:
maybe a default argument of `false` would be a little clearer for the `canContactCourseTeam` arg?